.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+
+# Change these settings to your own preference
+indent_style = space
+indent_size = 2
+
+# We recommend you to keep these unchanged
```

Comment:
Are these comments intentional? They look like they're from some example usage, they might be confusing to some people, making them think we don't enforce `space` and `2` (though `standard` does)